src/client/pages/OfferNew/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react'
+import { AnimateSharedLayout } from 'framer-motion'
+import styled from '@emotion/styled'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { Tab } from './Tab'
+
+const TabList = styled.div`
+  display: flex;
+  margin-bottom: 1.25rem;
+  margin-right: 1rem;
+  margin-top: 1rem;
+  max-width: 100%;
+  overflow-x: scroll;
+  -webkit-overflow-scrolling: touch;
```

Comment:
I think this has been removed from webkit. Do we really need it?

https://caniuse.com/mdn-css_properties_-webkit-overflow-scrolling